### DISCUSSION TITLE
#about points related to pull request

### DISCUSSION THREAD

**Discussion by nandani-goyal** [anuragverma108/SwapReads#2116] [Q&A]
Please can you resolve my one doubt. What if I solve 2 issues using one pull request only .Will I get score for both of them or not.

**Comment by DevGajjar28** [comment]
No, If you want both Issue score than do it separately creating a different branch and open PR.

**Comment by systemadmin-lab** [comment]
No, If you want both Issue score than do it separately creating a different branch and open PR.
